making-aeternity-wallet-transactions-on-android.md

```diff
@@ -0,0 +1,147 @@
+        userKeyPair = EncodingUtils.createBaseKeyPair(keyPairService.generateDerivedKey(mnemonicKeyPair, true))
+        Log.d("usersMnemonicSeedWord",mnemonicKeyPair.mnemonicSeedWords.joinToString{it+" "})
+        Log.d("usersKeys",userKeyPair.privateKey+"\n ${userKeyPair.publicKey}")
+    }
+ ```
+ one thing to note presently is that there is a slight bug with creating a keypair with a mnemonic, you can follow the issue on github here https://github.com/kryptokrauts/aepp-sdk-java/issues/126
+ ### Creating A KeyPair Without A Mnemonic String
+ ```kotlin
+ fun createAccountWithoutMnemonic(){
+  userKeyPair= keyPairService.generateBaseKeyPair()
+   Log.d("usersKeys",userKeyPair.privateKey+"\n ${userKeyPair.publicKey}")
+ }
+ ```
+ Then you can always recover the created Keypair by using the privateKey
+ ```kotlin
+  fun recoverWalletFromPrivateKey(){
+  var recoverredKeyPair= keyPairService.generateBaseKeyPairFromSecret(userKeyPair.privateKey);
+   Log.d("usersKeys",recoverredKeyPair.privateKey+"\n ${recoverredKeyPair.publicKey}")
+  }
+ ```
+ 
+ 
+ 
+ ## Transfer Tokens
+  Next we will create a function called ``sendAe`` that will receive as input two Strings, one for the publicAddress and the other for
+  the amount
+  ```kotlin
+    fun sendAe(var pAddrToSend:String,var amount:String){
+       var account = aeternityService.accounts.blockingGetAccount(Optional.of(pAddrToSend))
+        var spendTx = SpendTransactionModel.builder().amount(UnitConversionUtil.toAettos(amount, UnitConversionUtil.Unit.AE).toBigInteger()).nonce(account.nonce.add(
+            BigInteger.ONE)).recipient(
+           pAddrToSend).sender(userKeyPair.publicKey).ttl(BigInteger.ZERO).build()
+        var result = aeternityService.transactions.blockingPostTransaction(spendTx, userKeyPair.privateKey)
+    
+    }
+  ```
+  Notice how we use the privateKey field on the userKeyPair object to access the private key of the already created user so as to
+send tokens from his wallet  
+  
+  ## Checking Wallet Balance
+    Lastly we will create a function to get the balance of a user using his public address
+    
+```kotlin   
+fun getAeBalance(var publicAddress:String){
+
+        var account= aeternityService.accounts.blockingGetAccount(Optional.of(publicAddress))
+
+        Thread({
+            var account= aeternityService.accounts.blockingGetAccount(Optional.of(userKeyPair.publicKey));
+            Log.d("balance","Balance is ${account.balance}")
+        }).start()
+    }
+```
+ # Things To Note
+  - This tutorial was done using the gradle pugin 3.6.0 and Gradle build tools version 5.4.6
+  - BigDecimal and BigInteger are used instead of  double, float and int for a higher precision while making calculation
+  - It is important to remember to put most of this operations in another thread using Coroutines, Thread or AsyncTasks
+  - You can always check out the documentation here https://kryptokrauts.gitbook.io/aepp-sdk-java/use-the-sdk/overview-structure
```

Comment:
I would prefer referring to the main entrypoint of the documentation here in case the path in the url changes for some reason:
- https://kryptokrauts.gitbook.io/aepp-sdk-java